azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -510,6 +511,13 @@ private void addThreadPoolGauges(final QueuedThreadPool threadPool) {
   private void startWebMetrics() throws Exception {
     this.metricsManager
         .addGauge("WEB-NumQueuedFlows", this.executorManagerAdapter::getQueuedFlowSize);
+
+    // Metric for flows that have been submitted, but haven't started for more than 20 minutes.
+    if (this.props.getBoolean(ConfigurationKeys.AZKABAN_POLL_MODEL, false)) {
+      this.metricsManager
+          .addGauge("WEB-NumAgedQueuedFlows", this.executorManagerAdapter::getAgedQueuedFlowSize);
```

Comment:
I believe web server metrics are prefixed and grouped under "AZ-WEB" so adding "WEB" in the metric's name is somewhat redundant. Please, consider renaming this metric to  "num-aged-queued-flows" or similar.

Reply:
I am only following the convention that's already present in the file. I wouldn'y have settled for this convention to begin with. But at this point, it will lead to a mix.

Reply:
I understand there is currently a mix of naming conventions around metric names. I was hoping to follow the one applied to the majority of metrics in CommonMetrics.java and ExecMetrics.java further on and maybe at some point fix the legacy names.